docs/user/supported_devices.rst

```diff
@@ -101,7 +101,8 @@ ath79-generic
 
 * Ubiquiti
 
-  - NanoStation M2/M5 (XW)
+  - airMAX NanoBeam M5 (XW)
+  - airMAX NanoStation M2/M5 (XW)
```

Comment:
"airMAX" is something like a series/category name, but it isn't really a part of the model name (see https://www.ui.com/download/airmax-m for example)

Reply:
I see. Had loe dat the very same page to come to the conclusion, both `UniFi`and `airMAX` were series/category names looking at the navigation on the top instead of the left.

Will change it in a few minutes.

Reply:
resolved in 3e6c419